### ISSUE TITLE
Data Type Options Change each time starting the designer

### ISSUE THREAD

**Issue by ondras** [ondras/wwwsqldesigner#184]
_From [goo...@hawaiianhope.org](https://code.google.com/u/100408792243815598067/) on November 03, 2013 06:05:29_

I have been using your SQL Designer 2.6 now for about a year or so.  
Fantastic product ! What version of the product are you using? On what operating system? SQL Designer 2.6 on Internet Explorer 8 on Windows XP. What steps will reproduce the problem? I never really took noticed to this problem before, but today i can replicate it over and over.
1) close all browser.
2) Open a new browser
3) Go to : http://ondras.zarovi.cz/sql/demo/ 4) Create a new Table
5) Double Click on the field then Data Type:
   You will see either screen shot 1 or screen shot 2
   Take notice of the list of options for TYPE

6) close all browser.
7) Open a new browser
8) Go to : http://ondras.zarovi.cz/sql/demo/ 9) Create a new Table
10) Double Click on the field then on Data Type:
You may see screen shot 3 lots of additional options.  BUT notice that BLOB is now gone
Also in sc reen shot 4, notice that BIT is gone.

Any ideas ?
Basically for me to use this now i have to open and close it 3 times till i get the list from screen shot 3. What is the expected output? What do you see instead? See Screen Shot `#3` is the full list i expect to see all of the time. Please provide any additional information below. I downloaded your 2.7 version, but it appears to do the same thing a well.

**Attachment:** [1.jpg 2.jpg 3.jpg 4.jpg](http://code.google.com/p/wwwsqldesigner/issues/detail?id=177)

_Original issue: http://code.google.com/p/wwwsqldesigner/issues/detail?id=177_

**Comment by ondras** [comment]
_From [ondrej.zara](https://code.google.com/u/ondrej.zara/) on November 03, 2013 10:41:30_

When you open the Options dialog, what "Database for new designs" is selected? Your screenshots show an interesting mix of MySQL data types (#1) and MSSQL datatypes (#2, `#3`, `#4`...). Looks like the target database somehow gets changed during your workflow.

**Comment by ondras** [comment]
_From [goo...@hawaiianhope.org](https://code.google.com/u/100408792243815598067/) on November 03, 2013 13:09:16_

Originally it was the MySQL database.  When i would change it to MS SQL, it still would not chnage the data type options.  If i would close the browser and open it back up, then go back to the designer, then i would get the `#3` options.  This is not actually attached directly to any database, it is not interfacing with any, just using the design aspect.
I have since changed the configuration and set the default database type to MS SQL and it is now working as i would expect at startup.

**Comment by ondras** [comment]
_From [ondrej.zara](https://code.google.com/u/ondrej.zara/) on November 05, 2013 04:01:52_

Please note that the datatypes are not changed immediately (as indicated by an asterisk in the Options window), but rather only after you re-load the page. That might be causing your confusion.

**Status:** Done
